app/(tabs)/actor/[id].tsx

```diff
@@ -0,0 +1,541 @@
+import { Ionicons } from '@expo/vector-icons'
+import { Image } from 'expo-image'
+import { LinearGradient } from 'expo-linear-gradient'
+import { router, useLocalSearchParams } from 'expo-router'
+import { useMemo } from 'react'
+import {
+  ActivityIndicator,
+  ScrollView,
+  StyleSheet,
+  TouchableOpacity,
+  View,
+} from 'react-native'
+
+import { ThemedSafeAreaView } from '@/components/themed-safe-area-view'
+import { ThemedText } from '@/components/themed-text'
+import { Colors } from '@/constants/theme'
+import { useThemeColor } from '@/hooks/use-theme-color'
+
+// Mock data type - will be replaced with real API types later
+type MockActor = {
+  id: number
+  firstName: string
+  lastName: string
+  nickName?: string | null
+  birthdate?: string | null
+  birthPlace?: string | null
+  height?: string | null
+  occupations?: string[] | null
+  partners?: Array<{
+    name: string
+    period: string
+  }> | null
+  biography?: string | null
+  profile_image?: string | null
+  movies?: Array<{
+    id: number
+    title: string
+    image?: string
+    image_url?: string
+  }> | null
+}
```

Comment:
One more enhancement, you don't need to mock this type, let's use the `Actor` type, and add the `movies` prop as an optional property.